packs/classfeatures/avenger.json

```diff
@@ -79,28 +79,36 @@
                 "key": "ActiveEffectLike",
                 "mode": "upgrade",
                 "path": "flags.pf2e.favoredWeaponRank",
-                "predicate": [
-                    "item:slug:avenger"
-                ],
-                "value": 1
-            },
-            {
-                "key": "GrantItem",
-                "uuid": "Compendium.pf2e.actionspf2e.Item.Hunt Prey"
+                "value": "@actor.system.proficiencies.attacks.simple.rank"
             },
             {
                 "key": "ActiveEffectLike",
                 "mode": "upgrade",
                 "path": "system.proficiencies.defenses.medium.rank",
+                "value": "@actor.system.proficiencies.defenses.light.rank"
+            },
+            {
+                "category": "precision",
+                "diceNumber": "@actor.flags.pf2e.sneakAttackDamage.number",
+                "dieSize": "d{actor|flags.pf2e.sneakAttackDamage.faces}",
+                "key": "DamageDice",
+                "label": "PF2E.SpecificRule.SneakAttack",
                 "predicate": [
-                    {
-                        "gte": [
-                            "self:level",
-                            13
-                        ]
-                    }
+                    "target:condition:off-guard",
+                    "item:deity-favored"
                 ],
-                "value": "ternary(gte(@actor.system.proficiencies.defenses.light.rank,2),2,1)"
+                "selector": "strike-damage"
+            },
```

Comment:
This is how the Ruffian racket does Sneak Attack, but we can probably do better now by tagging weapons that qualify for Sneak Attack, predicating the main feature off of that, and then features that add or remove to the list can simply add or remove the tag. Not sure if we want to do that *now* though.

Reply:
How hard of a "lift" would it be to make the change? 
I think this way (tagging the item) is cleaner and easier to maintain.

Reply:
It's easy. I could probably do it in this PR, just not sure if there are stuff that would need migrating that I'm not considering.

Reply:
Fair